influxql.gen.go.tmpl

```diff
@@ -125,7 +143,7 @@ func (e *{{.name}}PointEmitter) EmitPoint() (models.Point, error) {
 		Time:       t,
 		Group:      e.group,
 		Dimensions: e.dimensions,
-		Tags:       e.tags,
+		Tags:       ap.Aux[0].(models.Tags),
```

Comment:
This looks like the cause of the CI failure. ap.Aux[0] will only have the tags if isSimpleSelector is true.